tests/py/dynamo/conversion/test_cdist_aten.py

```diff
@@ -0,0 +1,64 @@
+import torch
+import torch.nn as nn
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+
+from .harness import DispatchTestCase
+
+
+class TestCdistConverter(DispatchTestCase):
+    @parameterized.expand(
+        [
+            ((4, 3, 4), 0),
+            ((10, 3, 5, 2, 6), 0.5),
+            ((15, 10, 5), 0),
+            ((15, 10, 5), 0.5),
+            ((15, 10, 5), 1.0),
+            ((15, 10, 5), 1.5),
+            ((15, 10, 5), 2.0),
+            ((15, 10, 5), 2.99),
+            ((15, 10, 5), float("inf")),
+        ]
+    )
+    def test_cdist_float(self, shape, p):
+        class Cdist(nn.Module):
+            def forward(self, x1, x2):
+                print("x1 : ", x1)
+                print("x2 : ", x2)
```

Comment:
The print should be removed.

Reply:
Thanks! It seems I overlooked removing an unnecessary debug code